capstone-backend/pom.xml

```diff
@@ -62,6 +62,16 @@
       <artifactId>guava</artifactId>
```

Comment:
Remove Guava. We don't actually need it.

Reply:
Got it. Removed!